tests/dashbio_demos/utils/app_standalone.py

```diff
@@ -10,7 +10,7 @@ def run_standalone_app(
         filename
 ):
     """Run demo app (tests/dashbio_demos/app_*.py) as standalone app."""
-    app = dash.Dash(__name__, assets_folder='assets/')
+    app = dash.Dash(__name__, assets_folder='../../../assets/')
```

Comment:
```suggestion
    app = dash.Dash(__name__, assets_folder=os.path.join('..', '..', '..', 'assets/'))
```
I'm just being paranoid about Windows...

Reply:
@mkcor Good call! :) 
=